tasks/sshconfig.yml

```diff
@@ -29,8 +33,9 @@
   register: ssh_host_keys
   tags:
     - sshd
+    - always
```

Comment:
what does the `always` tags actually mean and when is it used?

Reply:
`- always` will ensure the play is run even if we run the playbook with "--tags " from the command line unless we specifically skip it.

Reply:
yeah, but the `always` tag is only used on SSH directories and files, and doesn't actually do anything except `register:`